### ISSUE TITLE
wmic not found

### ISSUE THREAD

**Issue by burnhamup** [Alex-D/check-disk-space#13]
Some of my users are getting an error when I run `check-disk-space`

```
Error invoking remote method 'download:start': Error: Command failed: wmic logicaldisk get size,freespace,caption
ERROR:

Description = The specified service does not exist as an installed service.
```

This sounds like they either don't have `wmic.exe` on their machine, or something has happened to their PATH.

I'm not sure if there's anything to be done here - I'm mainly opening the issue to see if other people are having the same problem. 

If corrupt PATHs are common, I could try looking up the EXE directly with `%SystemRoot%\System32\Wbem\wmic.exe`

**Comment by Alex-D** [comment]
Can you try the branch in PR #11 ?

**Comment by Topppy** [comment]
same problem on some windows7 .

**Comment by Alex-D** [comment]
Please see https://github.com/Alex-D/check-disk-space/pull/11#issuecomment-998669783

**Comment by jssuttles** [comment]
My team has been relying on WMIC as well. But it appears that it's being deprecated. https://stackoverflow.com/questions/57121875/what-can-i-do-about-wmic-is-deprecated

**Comment by lucas-barros** [comment]
WMIC is no longer available in Windows 11 dev channel (build 22483 or higher).

**Comment by Alex-D** [comment]
Should be fixed as of v3.2.0
